src/Illuminate/Support/Number.php

```diff
@@ -17,6 +18,136 @@ class Number
      */
     protected static $locale = 'en';
 
+    /**
+     * Get a new numeral instance for the given value.
+     *
+     * @param  int|float  $value
+     */
+    public static function of($value): Numeral
+    {
+        return new Numeral($value);
+    }
+
+    /**
+     * Determine if the given value is a number.
+     *
+     * @param  mixed  $number
+     */
+    public static function isNumeric($number): bool
+    {
+        return is_numeric($number);
+    }
+
+    /**
+     * Determine if the given value is an even number.
+     *
+     * @param  mixed  $value
+     */
+    public static function isEven($value): bool
+    {
+        return $value % 2 === 0;
+    }
+
+    /**
+     * Determine if the given value is an odd number.
+     *
+     * @param  mixed  $value
+     */
+    public static function isOdd($value): bool
+    {
+        return $value % 2 !== 0;
+    }
```

Comment:
Is `2.5` considered odd or even?

Should these methods be limited to integers?

Reply:
Fair question. Given that `2.5 / 2 === 1.25`, I'd say it's odd, but it can be limited to only integers if required. I can add tests for that use case if it's preferred to keep it as it is. I'll wait for a decision here